wire-ios/Wire-iOS Tests/Calling/CallGridViewControllerSnapshotTests.swift

```diff
@@ -148,7 +148,7 @@ final class CallGridViewControllerSnapshotTests: BaseSnapshotTestCase {
         createSut()
 
         // Then
-        verify(matching: sut)
+        verify(matching: sut, precision: 0.98, perceptualPrecision: 0.98)
```

Comment:
@caldrian This is also a temporary thing. I'm planning to create a ticket and investigate why CallGridViewController behaves weird sometimes.